setup.py

```diff
@@ -13,16 +13,16 @@
     version=__version__,
     description=('checks playbooks for practices and behaviour that could potentially be improved'),
     keywords='ansible, lint',
-    author='Will Thames',
-    author_email='will@thames.id.au',
-    url='https://github.com/willthames/ansible-lint',
+    author='Ansible by Red Hat',
```

Comment:
I'm not sure that this is the right thing to do. There's also `maintainer=` + `maintainer_email=` for this.
https://setuptools.readthedocs.io/en/latest/setuptools.html#metadata